### ISSUE TITLE
Publish `arm64` docker images

### ISSUE THREAD

**Issue by reisub** [nsmithuk/local-kms#37]
When I run the image published on dockerhub on an M1 Mac, I get the following output:

```
❯ docker run nsmithuk/local-kms
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
INFO[2022-03-30 14:56:22.026] Local KMS Version Unknown (Commit Hash Unknown)
INFO[2022-03-30 14:56:22.061] No file found at path /init/seed.yaml; skipping seeding.
INFO[2022-03-30 14:56:22.061] Data will be stored in /data
INFO[2022-03-30 14:56:22.061] Local KMS started on 0.0.0.0:8080
```

and when I build the image locally and run it:

```
INFO[2022-03-30 14:56:15.329] Local KMS Version Unknown (Commit Hash Unknown)
INFO[2022-03-30 14:56:15.335] No file found at path /init/seed.yaml; skipping seeding.
INFO[2022-03-30 14:56:15.335] Data will be stored in /data
INFO[2022-03-30 14:56:15.335] Local KMS started on 0.0.0.0:8080
```

I think it's just a matter of configuring the deployment pipeline to build multiple architectures and push them to dockerhub.

**Comment by InoMurko** [comment]
```
docker buildx build --platform linux/arm64/v8,linux/amd64,linux/386,linux/arm/v7 --push -t nsmithuk/local-kms:latest .
```

**Comment by nsmithuk** [comment]
Done. You now have a published ARMv8 / M1 Mac image on the tags `3`, `3.11.1` and `latest`.
